src/main/java/com/in28minutes/springboot/jdbc/h2/example/SpringBoot2JdbcWithH2Application.java

```diff
@@ -25,10 +25,14 @@ public static void main(String[] args) {
 	@Override
 	public void run(String... args) throws Exception {
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "index" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.2.7:9000/coding_rules#rule_key=squid%3AS1481)